apps/src/code-studio/components/libraries/LibraryClientApi.js

```diff
@@ -56,7 +57,7 @@ export default class LibraryClientApi {
     );
   }
 
-  fetchLatestVersionId(onSuccess, onError) {
+  fetchLatestVersionId(onSuccess, onError, event = 'unknown') {
     this.libraryApi.fetch(
       this.channelId + '/' + LIBRARY_NAME + '/versions',
```

Comment:
just for context: `this.channelId` is what is triggering this error, but i can't figure out how it is getting set to the library name rather than the actual channel ID. hopefully the context from the firehose logging can tell us the info we need to reproduce this error and track down why the channel ID is invalid